library/alloc/src/sync.rs

```diff
@@ -509,8 +520,11 @@ impl<T> Arc<T> {
     ///
     /// If `unwrap_or_drop` is called on every clone of this `Arc`,
     /// it is guaranteed that exactly one of the calls returns the inner value.
+    /// This means in particular that the inner value is not dropped.
+    ///
     /// The similar expression `Arc::try_unwrap(this).ok()` does not
-    /// offer this guarantee.
+    /// offer such a guarantee. See the last example below and the documentation
+    /// of `try_unwrap`[`Arc::try_unwrap`].
```

Comment:
```suggestion
    /// of [`Arc::try_unwrap`].
```

Reply:
A type, I actually wanted to write ``[`try_unwrap`][`Arc::try_unwrap`]`` which renders as `try_unwrap` but the link works. I thought there are enough mentions of the word `Arc` in this comment already.

Reply:
Yeah you can also put ``[`try_unwrap`][`Arc::try_unwrap`]``, its a function that's specific enough for it not to be confusing.

Reply:
> Yeah you can also put `` [`try_unwrap`][`Arc::try_unwrap`] ``, its a function that's specific enough for it not to be confusing.

And also, it’s fully spelled out in the previous sentence. That was the main reason I wanted it as short as possible.